### PR TITLE
Update glare to use faraday 1.0

### PR DESCRIPTION
**Pull request by joseluis-fw**

### CODE REVIEW

File: lib/glare.rb [hunk 1]
```diff
@@ -7,39 +9,49 @@
...
-      Domain.new(client).records(fqdn, type)
+      @client.records(fqdn, type)
     end
 
     private
 
-    CF_EMAIL = 'CF_EMAIL'.freeze
-    CF_AUTH_KEY = 'CF_AUTH_KEY'.freeze
-    CF_API_TOKEN = 'CF_API_TOKEN'.freeze
+    CF_EMAIL = 'CF_EMAIL'
```
Comment (pollosp) [inline lib/glare.rb]: Just curious why no more Freeze?

Comment (joseluis-fw) [reply lib/glare.rb]: Check the header of that file, mostly `rubocop` changes

File: lib/glare.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (pollosp) [inline lib/glare.rb]: Oh OK I see annotation here to remove freeze I didn't know this

Comment (josacar) [reply lib/glare.rb]: :+1: :joy:

**Pull request merged**